test/over_react/component/error_boundary/shared_stack_tests.dart

```diff
@@ -27,17 +29,15 @@ void sharedErrorBoundaryStackTests() {
     void expectRenderErrorWithComponentName(ReactElement element,
         {required String expectedComponentName}) {
       final capturedInfos = <ReactErrorInfo>[];
-      expect(() {
-        rtl.render((ErrorBoundary()
-          ..shouldLogErrors = false
-          ..onComponentDidCatch = (error, info) {
-            capturedInfos.add(info);
-          })(element));
-        // Use prints as an easy way to swallow `print` calls and
-        // prevent RTL from forwarding console errors to the test output,
-        // since React error boundary logging is pretty noisy.
-        // TODO instead, disable logging in this rtl.render call once that option is available: FED-1641
-      }, prints(anything));
+      final mountNode = DivElement();
+      // Use react_dom.render instead of RTL to avoid errors on React 18 about
+      // `act` being used in prod builds.
```

Comment:
Is this being addressed in a separate RTL-related React 18 prep ticket?

Reply:
@greglittlefield-wf is working on RTL prep - I'm not sure if this is something that would be addressed or if it's more like RTL shouldn't be used in prod builds in general and this is an edge case where a test is purposely trying to use a prod build?

Reply:
> it's more like RTL shouldn't be used in prod builds in general and this is an edge case where a test is purposely trying to use a prod build?

Yeah I'd lean toward that, and don't think we need to try to address anything in RTL unless this comes up again